prototype/southbound/device-factory/device-factory-core/src/test/java/org/eclipse/sensinact/gateway/southbound/device/factory/impl/BulkDTOPromise.java

```diff
@@ -0,0 +1,117 @@
+/*********************************************************************
```

Comment:
Is there a reason not to just use Promises.resolved(dto) and remove this type altogether?

Reply:
I reused some previous code. I'll change it in the 3 device factory projects.